eclair-core/src/main/resources/reference.conf

```diff
@@ -212,6 +212,7 @@ eclair {
           channel-age = 0.4       // when computing the weight for a channel, consider its AGE in this proportion
           channel-capacity = 0.55 // when computing the weight for a channel, consider its CAPACITY in this proportion
         }
+        use-ratios = true
```

Comment:
A boolean is a bit limiting, it only allows choosing between two options.
What about having one sub-section here per cost formula?
And one enum field that selects which "formula" to use?

It would look something like:

```
scoring-heuristic = "ratios" // can be ratios, failure-cost or some-other-way-of-scoring

ratios {
  // contains all parameters for scoring based on ratios
}

failure-cost {
  // contains all parameters for scoring based on wasted capital in case of failure
}

some-other-way-of-scoring {
  // contains parameter for some other way of computing a score
}
```

This feels more future-proof: it makes it easy to introduce widely different scoring formulas/heuristics.

Reply:
In the long term I don't think we should support several scoring formulas. The idea here is to try a different formula, get some numbers and then delete the worst formula.

Reply:
Ok, interesting, in that case we can do something more ad-hoc like you did